backend/app/api/endpoints/openapi_responses.py

```
@@ -145,7 +148,591 @@ def _task_to_response_object(
+        )
+
+    bot_ids = []
+    for member in team_crd.spec.members:
+        member_bot = (
+            db.query(Kind)
+            .filter(
+                Kind.user_id == team.user_id,
+                Kind.kind == "Bot",
+                Kind.name == member.botRef.name,
+                Kind.namespace == member.botRef.namespace,
+                Kind.is_active == True,
```

Ruff (0.14.10)

Avoid equality comparisons to `True`; use `Kind.is_active:` for truth checks

Replace with `Kind.is_active`

(E712)